src/pages/AdminManageUsers.js

```diff
@@ -123,6 +123,10 @@ export const AdminManageUsers = function AdminManageUsers() {
           a({
             id: 'btn_addUser',
             className: 'btn-primary btn-add common-background',
+            style: {
+              marginTop: '30%',
+              display: 'flex'
+            },
```

Comment:
When looking at other Admin Console pages, I noticed ManageDac had style inputs for the "Add DAC" button. When testing things out I noticed that adding this for the AdminManageUsers page helped to bring the "Add User" text to the center of the button.

Without this change, the text is falling off of the bottom of the button - wasn't sure if that was a bug or not. I did a quick search and didn't see any tickets relating to updating styling on this button so I just included it here. Please let me know if I should remove this change due to it not being in the scope of this ticket.